src/xmipp/libraries/tomo/tomo_extract_subtomograms.cpp

```diff
@@ -42,8 +41,10 @@ void ProgTomoExtractSubtomograms::readParams()
 	invertContrast = checkParam("--invertContrast");
 	normalize = checkParam("--normalize");
 	fnOut = getParam("-o");
+	downsample = checkParam("--downsample");
```

Comment:
We should avoid to have two parameter with the same name. Let's call it downsamplingFactor, or that only will be used if --downsampling is passed

Reply:
I do not understand. Only one input param is called dowsample. Its presence is checked and saved in the boolean downsample variable, if present the value is saved in the downsampleFactor.

Reply:
Fixed :)